src/Automation/app.config

```diff
@@ -6,22 +6,9 @@
         <assemblyIdentity name="Newtonsoft.Json" publicKeyToken="30ad4fe6b2a6aeed" culture="neutral" />
         <bindingRedirect oldVersion="0.0.0.0-10.0.0.0" newVersion="10.0.0.0" />
       </dependentAssembly>
-      <dependentAssembly>
-        <assemblyIdentity name="System.Net.Http.Formatting" publicKeyToken="31bf3856ad364e35" culture="neutral" />
```

Comment:
>Http [](start = 43, length = 4)

System.Net and the Microsoft.IdentityModel assemblies were originally there for the ADO support. We don't need them for anything in axe-windows